### ISSUE TITLE
Add autodeploy infrastructure

### ISSUE THREAD

**Issue by arrdem** [jaunt-lang/jaunt#17]
Since testing is already covered by #3, it only makes sense to have deployment managed automagically by tooling as well. Maybe this is a job for jitpack, maybe it's something I can piggyback off of circleci for.

**Comment by arrdem** [comment]
This is pretty clearly something that CircleCI can be made to do, [1](https://circleci.com/docs/configuration#deployment). The trick is going to be figuring out on our end how to manage it and where we're deploying to. `mvn deploy` seems to require a well placed boot from preliminary examination.

**Comment by arrdem** [comment]
One step in this direction which isn't PR ready.. waiting on #30 

[0001-Emit-jars-to-CircleCI-s-ARTIFACTS-dir.txt](https://github.com/arrdem/clojarr/files/106171/0001-Emit-jars-to-CircleCI-s-ARTIFACTS-dir.txt)

**Comment by arrdem** [comment]
This is mostly working now.
- `master` patches will be deployed blindly as releases.
- `develop` patches are deployed as `-SNAPSHOT` releases.
- `vA.B.C(-\W+)?` tagged objects are also deployed blindly as releases ([build 323](https://circleci.com/gh/jaunt-lang/jaunt/323))

Maybe there's something left to be done so that pull requests also push qualified snapshots, but I'm really not sure.

**Comment by arrdem** [comment]
Live as of #53
